ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -454,11 +512,6 @@ protected static void endToEndReplicationWithMultiNodeMultiPartitionTest(int int
           }
 
           // get user metadata
-          ids.clear();
```

Comment:
these lines of code are no longer required?

Reply:
yes. check lines 495 to 498. Felt its unncessary, hence removed it